google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -73,7 +73,7 @@
   private static final String MANIFEST_ARTIFACT_ID_KEY = "artifactId";
   private static final String MANIFEST_VERSION_KEY = "Implementation-Version";
   private static final String ARTIFACT_ID = "google-cloud-core";
-  private static final String LIBRARY_NAME = "gcloud-java";
```

Comment:
So this PR is going to inject the 'gccl' header for Veneer/Vtk clients. I will submit a separate PR for Veneer only clients.
Are you suggesting we use 'gccl' in the x-google-api-client header and 'gcloud-java' in user agent? Wouldn't that be inconsistent and confusing?